types/merkle/README.md

```diff
@@ -0,0 +1,16 @@
+# Installation
```

Comment:
The file will be generated automatically when published to npm, so you don't need to add this file as source code.